pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -538,9 +691,18 @@ func afwSchedulerEntry(client splcommon.ControllerClient, cr splcommon.MetaObjec
 
 	var clusterScopedApps []*enterpriseApi.AppDeploymentInfo
 
+	afwEntryTime := time.Now().Unix()
+
+	// get the current available disk space for downloading apps on operator pod
+	availableDiskSpace, err := getAvailableDiskSpace()
+	if err != nil {
+		return err
+	}
+	afwPipeline.availableDiskSpace = availableDiskSpace
+
 	// Start the download phase manager
 	afwPipeline.phaseWaiter.Add(1)
-	go afwPipeline.downloadPhaseManager()
+	go afwPipeline.downloadPhaseManager(appDeployContext)
```

Comment:
Let us have the reference for appDeployContext,  at the overall pipeline level, so that other phases also can have access to it. Other PR is covering this change.